packages/audits/src/components/AuditsRawTable.tsx

```diff
@@ -42,13 +42,28 @@ export const AuditsRawTable: FC<IAuditsRawTable> = React.memo(({ headerProps, ..
       );
   }, [headerProps]);
 
+  const getTableData = useCallback(() => {
+    const columnsName = headerProps.map(({ name }) => name);
+    let tableData = [...data];
+
+    for (const columnName of columnsName) {
```

Comment:
why do you run that check for all of the columns types and not only for JSON?

Reply:
because right now column types can be changed (delete column and add it again with another type)